Makefile

```diff
@@ -224,7 +224,7 @@ test/profiler: $(GO_SRC) $(LIBBPF_HEADERS) $(LIBBPF_OBJ) bpf
 	sudo $(GO_ENV) $(CGO_ENV) $(GO) test $(SANITIZERS) -v ./pkg/profiler/... -count=1
 
 .PHONY: test/integration
-test/integration: test/integration/native test/integration/python test/integration/ruby
+test/integration: test/integration/native test/integration/python test/integration/ruby test/integration/nodejs
```

Comment:
Probably, this needs rebasing I just merged another PR that also touches here.

Reply:
OK